design-system/docs/app/(site)/layout.tsx

```diff
@@ -28,7 +28,14 @@ export default async function RootLayout({
 }) {
   return (
     <NextRootProvider fontClassName={inter.variable}>
-      <head>{nextRootScript}</head>
+      <head>
+        {nextRootScript}
+        <style
+          dangerouslySetInnerHTML={{
+            __html: /* css */ `html, body { height: 100%; }`,
```

Comment:
This is so the main content takes up available space so we can push the footer to the bottom of the page without having to rely on viewport units


| Before                                                                                     | After                                                                                     |
|--------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
| ![before](https://github.com/user-attachments/assets/ea92f509-0000-4e56-86f0-ca8d905c885f) | ![after](https://github.com/user-attachments/assets/5dc2a91b-1b8e-4069-91eb-d1ae3b63bcf1) |